app/CSVManager.py

```diff
@@ -14,19 +14,23 @@ def __init__(self, spark_session, data_directory_path):
         self.__spark_session = spark_session
         self.__data_directory_path = data_directory_path
 
-    def read(self, path_in_data_dir, header=True):
+    def read(self, path_in_data_dir, schema=None, header=True):
         """Reads PySpark dataframe from csv file from data_directory_path.
 
         Args:
             path_in_data_dir (string): path to csv file or directory with csv files in data_directory_path.
             header (bool): does csv have header as first row.
+            schema: schema for dataset
```

Comment:
docstring and declaration does not match each other. please move argument description upper or change order of paramenters of class method. it would be also nice to have data type in parenthesis.